src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -26,12 +26,24 @@ export const AppContainer = ({ view, params }: { view?: View; params: AppMountPa
       {/* TODO: improve fallback state */}
       <Suspense fallback={<div>Loading...</div>}>
         <Context {...params}>
-          <Sidebar>
-            <Panel {...params} />
-          </Sidebar>
-          <EuiPageBody className="deLayout__canvas">
-            <Canvas {...params} />
-          </EuiPageBody>
+          <EuiResizableContainer>
+            {(EuiResizablePanel, EuiResizableButton) => (
+              <>
+                <EuiResizablePanel initialSize={140} minSize="10%" mode="collapsible">
```

Comment:
`initialSize` is number in percent per OUI documentation, could you check if `initialSize={140}` and `initialSize={1140}` are correctly set?

Reply:
Thanks @ruanyl , Updated the initial size with percent according to OUI documentation.